jlm/hls/util/view.cpp

```diff
@@ -19,6 +18,29 @@
 namespace jlm::hls
 {
 
+std::string
+ViewcolorToString(const ViewColors & color)
+{
+  switch (color)
+  {
+  case NONE:
+    return "";
+    break;
+
+  case BLACK:
+    return "black";
+    break;
+
+  case RED:
+    return "red";
+    break;
+
+  default:
+    JLM_UNREACHABLE("HLS view color not defined");
+    break;
+  }
```

Comment:
Why not getting rid of the NONE color and just returning `""` be default?

Reply:
Still need to define something representing the option in the enum, so then prefer to be explicit.
The default will only be triggered if one add another color to ViewColors and doesn't update this function.